src/domain/hanke/list/HankeList.tsx

```diff
@@ -19,34 +19,38 @@ const getProjects = async () => {
   const data = await api.get(`/hankkeet/`);
   return data;
 };
+
 const useProject = () => useQuery(['project'], getProjects);
-const Projects: React.FC = () => {
+// eslint-disable-next-line
+const Projects: React.FC<any> = ({ fakeData }) => {
```

Comment:
Add better types. Rename fakeData to initialData and pass initialData to react-query: https://react-query.tanstack.com/guides/initial-query-data

Reply:
Or maybe split data fetching to separate component.